sproxy.go

```diff
@@ -0,0 +1,189 @@
+// Copyright 2017 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package sproxy
+
+import (
+	"errors"
+	"sync"
+	"time"
+)
+
+type Event struct {
+	StatusCode   int       `json:"code,omitempty"`
```

Comment:
Do we need the JSON tags? If not, can we remove them?

Reply:
I added them for inspection and thought perhaps the actual StackDriverMonitoring uploading code might need JSON marshaled content. But sure I can remove them until we need them.